ci/azure-pipeline-security-cloudfoundrysinglesignon.yaml

```diff
@@ -1,18 +0,0 @@
-trigger:
```

Comment:
Please update `/README.md`, which contains a reference to the old "CloudFoundrySingleSignon" sample.